bundle/compliance/cis_azure/rules/cis_7_4/test.rego

```diff
@@ -3,10 +3,10 @@ package compliance.cis_azure.rules.cis_7_4
 import data.compliance.policy.azure.data_adapter
 import data.lib.test
 
-generate_encryption_settings(type) = {
+generate_encryption_settings(type) = {"encryption": {
```

Comment:
Unrelated to 7.2, just wanted to make sure 7.4 handles missing "encryption" data